app/src/main/java/io/appium/uiautomator2/server/ServerConfig.java

```diff
@@ -16,10 +16,131 @@
 
 package io.appium.uiautomator2.server;
 
+import java.util.HashMap;
+import java.util.Map;
+
+import io.appium.uiautomator2.model.settings.MjpegBilinearFiltering;
+import io.appium.uiautomator2.model.settings.MjpegScalingFactor;
+import io.appium.uiautomator2.model.settings.MjpegServerFramerate;
+import io.appium.uiautomator2.model.settings.MjpegServerPort;
+import io.appium.uiautomator2.model.settings.MjpegServerScreenshotQuality;
+import io.appium.uiautomator2.model.settings.ServerPort;
+
 public class ServerConfig {
-    private final static int PORT = 6790;
+    // server port (default: `6790`)
```

Comment:
These comments are redundant

Reply:
This will be resolved w/ the upcoming update(s)